diagnostic.md

```diff
@@ -32,23 +34,29 @@ Give two examples of jQuery getter methods and two examples of jQuery setter
 methods.
 
 <!-- your answer starts here -->
-
+getter:
+.val()
+.html
+setter:
+.val(value)
+.html(value)
 <!-- your answer ends here -->
 
 ## Question 5
 
 In your own words, please answer the following question: WHAT IS THE DOM?
 
 <!-- your answer starts here -->
-
+The DOM is a copy of code of a website to which temporary changes can be applied
```

Comment:
The DOM is a language agnostic API for representing HTML documents as a tree structure.  A web browser implements this interface to render web pages for you to view and interact with.